src/Kestrel.Transport.Sockets/Kestrel.Transport.Sockets.csproj

```diff
@@ -4,11 +4,12 @@
     <AssemblyName>Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets</AssemblyName>
     <RootNamespace>Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets</RootNamespace>
     <Description>Managed socket transport for the ASP.NET Core Kestrel cross-platform web server.</Description>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0;netcoreapp2.1</TargetFrameworks>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <PackageTags>aspnetcore;kestrel</PackageTags>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <NoWarn>CS1591;$(NoWarn)</NoWarn>
+    <EnableAPICheck>false</EnableAPICheck>
```

Comment:
I disabled this for 2 reasons:
- We haven't shipped this package yet.
- API check is broken for netcoreapp2.1 projects. I submitted a PR for this.

Reply:
This PR didn't change anything APICheck would complain about, right? Either way, I created #2175 to track reenabling this once we ship.